src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -474,6 +485,18 @@ static ConfigDef newConfigDef() {
             5,
             ConfigDef.Width.NONE,
             INGESTION_METHOD_OPT)
+        .define(
+            SNOWPIPE_STREAMING_FILE_TYPE,
+            Type.STRING,
+            SNOWPIPE_STREAMING_FILE_TYPE_DEFAULT,
+            BDEC_VERSION_VALIDATOR,
```

Comment:
I don't think a validator is needed, we already have validation logic in the SDK, it will throw an exception if it's something else

Reply:
Sounds good, will keep a string validator though.